action.yml

```
@@ -58,6 +58,10 @@ runs:
         cache: ${{ inputs.poetry_cache_enabled == 'true' && 'poetry' || ''}}
         cache-dependency-path: ${{ inputs.poetry_cache_enabled == 'true' && format('{0}/poetry.lock', inputs.working_directory) || ''}}
 
+    - run: python ${{ github.action_path }}/suspicious-package-versions.py
```

```suggestion
    - run: python ${{ github.action_path }}/invalid-package-versions.py
```
Or something other than `invalid`. To me `suspicious` suggests that this is some sort of security check which it is not.

---

What about `misconfigured`?

---

I like invalid.

---

Changed it to invalid.